### PR TITLE
Feat: signup

### PR DESCRIPTION
**Pull request by nyyshaaa**
https://deploy-preview-2--adoring-hawking-f016bc.netlify.app/pages/sign-up/signup.html

### CODE REVIEW

File: pages/sign-up/signin.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline pages/sign-up/signin.html:7]: add author meta tag.

**Review by yashpatel1797 — APPROVED** [review]
Design is clean. code looks good to me.

File: pages/sign-up/signup.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
+.grids-wrapper>*{
+    color:black;
+    /* background-color: var(--greyLight-1); */
+    box-sizing: border-box;
```
Comment (Rockyraven) [inline pages/sign-up/signup.css:2]: remove unwanted code and extra space

File: pages/sign-up/signup.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    width: 15rem;
+    text-align: center;
+    padding-bottom: 1rem;
+    border-bottom: 2px solid var(--gold1);
+    font-size: 2rem;
+    font-weight: 600;
+   
+    
+}
+
```
Comment (Rockyraven) [inline pages/sign-up/signup.css:53]: remove space here also

File: pages/sign-up/signup.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    font-size: 1.8rem;
+    font-weight: 500;
+}
+
+.text-input{
+    margin-bottom: 1.6rem;
+    border: none;
+    outline-color: var(--gold1);
+    /* outline-color: var(--white); */
+    height: 3rem;
```
Comment (Rockyraven) [inline pages/sign-up/signup.css:80]: remove unwanted code

File: pages/sign-up/signup.css [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+.tab-wrapper{
+    height: 100%;
+    flex-basis: 40rem;
+    background-color: var(--pink8);
+    color: var(--gold1);
+    box-shadow:  5px 5px 12px #120f1d,
+    -1px -1px 12px #141120;
+    
+}
```
Comment (Rockyraven) [inline pages/sign-up/signup.css:43]: here var color should be better

File: pages/sign-up/signin.html (hunk 1 above)
Comment (nyyshaaa) [reply pages/sign-up/signin.html:7]: okeiy i will add it.Thank You!

File: pages/sign-up/signup.css (hunk 5 above)
Comment (nyyshaaa) [reply pages/sign-up/signup.css:43]: you mean to put box-shadow in var. okeiy i missed maybe i did for some ,will do.

File: pages/sign-up/signup.css (hunk 2 above)
Comment (nyyshaaa) [reply pages/sign-up/signup.css:2]: okeiy

**Pull request merged**